go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/sigstore/helm-sigstore
 
-go 1.22
+go 1.22.0
```

Comment:
i think here we can bump to 1.23 as this is not being used as a lib or in other downstream